README.md

```diff
@@ -543,6 +543,19 @@ export NODE_BINARY=node
 ../bin/react-native-xcode.sh
 ```
 
+# Open container app
+Steps needed to open the host application from the share extension.
+1) Allow your app to be opened via URL Scheme - [Learn more](https://medium.com/react-native-training/deep-linking-your-react-native-app-d87c39a1ad5e)
+2) In xcode, select share extension and go to Build Settings and set **Require Only App-Extension-Safe API** to `NO`.
```

Comment:
@birkir @NikitaPokryshko What are the potential risks/concerns with flipping this value?

Reply:
Your app may be rejected, in two apps I've published it's was approved, So I am not sure.

Reply:
I'll be submitting an app to the app store using this fork/approach in about a week's time, can update here if another data point is useful.

Reply:
Sorry for the late update - Apple accepted my recent publication with this included.